app/src/main/java/org/fossasia/openevent/general/speakers/SpeakerWithEventDao.kt

```diff
@@ -0,0 +1,23 @@
+package org.fossasia.openevent.general.speakers
+
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+import androidx.room.RoomWarnings
+import io.reactivex.Single
+
+@Dao
+interface SpeakerWithEventDao {
+    @SuppressWarnings(RoomWarnings.CURSOR_MISMATCH)
+    @Query("""
+        SELECT * FROM speaker
+        INNER JOIN speakerwithevent ON
+        speaker.id = speakerwithevent.speaker_id
+        WHERE speakerwithevent.event_id = :eventID
+        """)
+    fun getSpeakerWithEventId(eventID: Long): Single<List<Speaker>>
```

Comment:
```suggestion
    fun getSpeakerWithEventId(eventID: Long): LiveData<List<Speaker>>
```

Reply:
@iamareebjamal LiveData ? then i would have to return live data instead of single